civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -454,6 +454,15 @@ function civihr_employee_portal_update_7035() {
   drush_civihr_employee_portal_refresh_node_export_files();
 }
 
+/**
+ * Disables the HR Vacancies View
+ */
+function civihr_employee_portal_update_7036() {
+  $status = variable_get('views_defaults', array());
+  $status['hr-vacancies'] = TRUE;
+  variable_set('views_defaults', $status);
+}
+
```

Comment:
I guess we probably need to remove this also: https://github.com/compucorp/civihr-employee-portal/blob/65650c5f4adc715ffc49336204131ad1136c7c68/civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.info#L61